dm/syncer/syncer.go

```diff
@@ -3764,3 +3798,55 @@ func calculateChanSize(queueSize, workerCount int, compact bool) int {
 	}
 	return chanSize
 }
+
+func (s *Syncer) setGlobalPointByTime(tctx *tcontext.Context, timeStr string) error {
+	// we support two layout
+	t, err := time.ParseInLocation("2006-01-02 15:04:05", timeStr, s.timezone)
```

Comment:
there is too many magic string `2006-01-02 15:04:05` in our code now 😂 maybe add a const insted
please also add a const for `2006-01-02T15:04:05`